h2/src/test/org/h2/test/store/TestMVStore.java

```diff
@@ -1416,7 +1416,9 @@ private void testFastDelete() {
             assertEquals(i + 1, m.size());
         }
         assertEquals(1000, m.size());
-        assertEquals(131896, s.getUnsavedMemory());
+        // previously (131896) we fail to account for initial root page for every map
+        // there are two of them here (meta and "data"), hence lack of 256 bytes
+        assertEquals(132152, s.getUnsavedMemory());
```

Comment:
would also be good to note in commit message that you have fixed a flaw in normal memory accounting